proto/warden/warden/v1beta3/tx.proto

```diff
@@ -153,6 +153,24 @@ message MsgUpdateKeychain {
 
 message MsgUpdateKeychainResponse {}
 
+message MsgAddKeychainAdminRequest {
+  option (cosmos.msg.v1.signer) = "authority";
+  string authority = 1;
+  uint64 keychain_id = 2;
+  string new_admin = 3;
```

Comment:
```suggestion
  string new_admin = 3 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

we should start using this annotation for strings that are addresses, at some point we'll have to fix all the other places too 😅